### PR TITLE
The Inner Source Learning Path

### PR DESCRIPTION
**Pull request by rrrutledge**
* Proposal for OSCON 2019.
* View the [rendered proposal](https://github.com/rrrutledge/talks/blob/oscon2019/learning-path/2019/OSCON/the-inner-source-learning-path/proposal.md).

### CODE REVIEW

**Comment by rrrutledge** [comment]
Especially looking for feedback from @Silona, @gruetter, and @Danese

File: 2019/OSCON/the-inner-source-learning-path/proposal.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Learn key roles and principles for effective inner source and how you can share them with others.
+
+# Topic
+
+Sustainable Open Source
+
+# Abstract
+
+Inner source is the application of open source methodologies to company-internal software.
```
Comment (gruetter) [inline 2019/OSCON/the-inner-source-learning-path/proposal.md]: If I were Klaas, my comment would be: s/methodologies/methods. _Methodology_ is the field of study of methods. I think he has a point.

File: 2019/OSCON/the-inner-source-learning-path/proposal.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* short introduction to inner source.
+* explanation of the problems that inner source can solve.
+* overview of the mechanics and roles of inner source.
+* review of the key benefits of inner source.
+* deeper look at the key roles of inner source.
+  * Product Owner
+  * Trusted Committer
+  * Contributor
+* summary of concepts and links to the online learning path resources.
+
```
Comment (gruetter) [inline 2019/OSCON/the-inner-source-learning-path/proposal.md:40]: I don‘t know if we‘ll eventually add another LP videos in the future, but wouldn‘t it be nice to call on InnerSource practitioners to share their learnings in a video (on our YouTube Channel) and for newcomers to ask questions they would like to answer? Also, I think this talk would be a great opportunity to outline the other channels we provide to share knowledge: Patterns, Books, Summits. I‘d mention the whole range here.

**Review by gruetter — CHANGES_REQUESTED** [review]

File: 2019/OSCON/the-inner-source-learning-path/proposal.md (hunk 1 above)
Comment (rrrutledge) [reply 2019/OSCON/the-inner-source-learning-path/proposal.md]: Updated.

File: 2019/OSCON/the-inner-source-learning-path/proposal.md (hunk 2 above)
Comment (rrrutledge) [reply 2019/OSCON/the-inner-source-learning-path/proposal.md:40]: This sounds good to include, but maybe too detailed and loosely coupled to include in this CFP proposal?  Thinking of keeping things simple here so the reviewers can clearly understand the focused thrust of our content and accept it.  Once we're "in" I think that we can include some of those other pieces of information that you bring up in the actual presentation at OSCON.

**Review by rrrutledge — COMMENTED** [review]
Thanks for the review, Georg!

**Review by gruetter — APPROVED** [review]

**Pull request merged**